### PR TITLE
GitHub Issue NOAA-EMC/GSI#272. A fix for the corrupted final statistics written out in the GSI FV3-LAM parallel IO for cold start runs

### PR DESCRIPTION
**Pull request by TingLei-daprediction**
A quick fix for the corrupted final statistics in files like fort.201. See GSI ISSUE https://github.com/NOAA-EMC/GSI/issues/272.

### CODE REVIEW

**Comment by TingLei-daprediction** [comment]
In the fix, when fv3sar_bg_opt=1, an temporary allocatable array is created to contained the surface pressure in unit Pa while the original ges_ps is not changed .

**Comment by MichaelLueken-NOAA** [comment]
@TingLei-daprediction Who would you like to have review these changes?

**Comment by TingLei-daprediction** [comment]
I think that @guoqing-noaa and @hu5970 would be appropriate to review this quick fix. 
Thanks.

File: src/gsi/gsi_rfv3io_mod.f90 [hunk 1]
```diff
@@ -69,6 +69,7 @@ module gsi_rfv3io_mod
 
   real(r_kind),dimension(:,:  ),allocatable:: ges_ps_bg 
   real(r_kind),dimension(:,:  ),allocatable:: ges_ps_inc 
+  real(r_kind),dimension(:,:  ),allocatable:: ges_ps_write
```
Comment (hu5970) [inline src/gsi/gsi_rfv3io_mod.f90:72]: ges_ps_write is used only in subroutine wrfv3_netcdf. Should it be defined inside that subroutine?

Comment (TingLei-daprediction) [reply src/gsi/gsi_rfv3io_mod.f90]: Changed as suggested

**Review by hu5970 — APPROVED** [review]
Tested and the fix works well.

**Comment by MichaelLueken-NOAA** [comment]
Hi @TingLei-daprediction, I have completed my visual review and the changes look good.  I cloned your fork and was able to compile and run the tests.  Following yesterday's master update, there were differences.  After merging the authoritative master to your changes, all tests passed.  I compiled the code in debug mode and no new warnings or errors were encountered.  All debug tests successfully ran through to completion.  Before I submit your work to the review committee, please update your master to the latest authoritative master using:

1) git remote add upstream https://github.com/NOAA-EMC/GSI.git
2) git remote update
3) git rebase upstream/master
4) git submodule update
5) git commit --amend
6) git push origin master --force (please save and close commit window without changing commit message)

Once complete, I will submit your changes to the review committee.  If you have any questions, please let me know.

**Comment by TingLei-daprediction** [comment]
Mike,
Thanks a lot as well as for the details instructions.
I had completed those steps.
Please let me if any problems.
Regards,
Ting

**Comment by MichaelLueken-NOAA** [comment]
The DA review committee's due date has passed with no feedback, so I will now give final approval to these changes and merge them to the authoritative repository.

**Pull request merged**